docs/docs-content/tenant-settings/add-registry.md

```diff
@@ -10,79 +10,14 @@ tags: ["enterprise", "management", "registry"]
 
 You can add a registry at the system level or tenant level. Registries added at the system level are available to all the tenants. Registries added at the tenant level are available only to that tenant. This section describes how to add a tenant-level registry. For guidance on adding a registry at the system scope, check out [Add System-Level Registry](../enterprise-version/system-management/add-registry.md). 
```

Comment:
Maybe we can clarify that you can only add a registry at the system level if you have self-hosted palette? this paragraph reads a bit like someone using SaaS Palette can also do this. Although the page that we refer them to will answer the question, maybe a new user would feel a bit disoriented. 

I see that you have a paragraph later on this page that clarifies this, maybe we can move that language up a bit.